backend/src/main/java/se/kth/iv1201/recruitmentapplicationgroup5/controller/ErrorMsgBody.java

```diff
@@ -0,0 +1,12 @@
+package se.kth.iv1201.recruitmentapplicationgroup5.controller;
+
+import lombok.Value;
+
+@Value
+public class ErrorMsgBody {
```

Comment:
I think that the field status and URL is a good idea to have in all of our response bodies. (Side note)

Reply:
Yes. Maybe even message, maybe this should be the generic responsebody that other responses inherit. Maybe add an "Object payload" field, and it can deliver the payload, and "String[] links".